packages/pwa-kit-react-sdk/src/ssr/browser/main.jsx

```diff
@@ -71,7 +71,7 @@ const OuterApp = ({routes, error, WrappedApp, locals}) => {
 OuterApp.propTypes = {
     routes: PropTypes.array.isRequired,
     error: PropTypes.object,
-    WrappedApp: PropTypes.element.isRequired,
+    WrappedApp: PropTypes.func.isRequired,
```

Comment:
So are we moving from `WrappedApp` being an `element` to being a SFC (stateless functional component)?

Reply:
It is a HOC actually, the wrappedApp is supposed to be a RouteComponent what is wrapped with some [HOC](https://github.com/SalesforceCommerceCloud/pwa-kit/blob/develop/packages/pwa-kit-react-sdk/src/ssr/universal/components/route-component/index.js#L399).